test/surf-extraction.Tests.ps1

```diff
@@ -66,6 +75,31 @@ The file supplied is not a valid extractor configuration file.
         Get-Extractors | Format-MultilineOutput | Should -Not -Match $extractorName
     }
 
+    It "should be created from a list of module IDs " {
+        New-ExtractorFromModules $extractorName "waives.name" "waives.date" | Format-MultilineOutput | Should -Be @"
+Creating extractor '$extractorName'... [OK]
+"@
+        $LASTEXITCODE | Should -Be 0
+
+        # Verify
+        Get-Extractors | Format-MultilineOutput | Should -Match $extractorName
+    }
+
+    It "should be created from a template" {
+        $extractorTemplate = (Join-Path $PSScriptRoot "extractor-template.json")
+        New-ExtractorFromTemplate $extractorName $extractorTemplate | Format-MultilineOutput | Should -Be @"
+Creating extractor '$extractorName'... [OK]
+"@
+        $LASTEXITCODE | Should -Be 0
+
+        # Verify
+        Get-Extractors | Format-MultilineOutput | Should -Match $extractorName
+    }
+
+    It "list available modules" {
+        Invoke-App list modules 2>&1 | Format-MultilineOutput | Should -Match "waives.date"
```

Comment:
✋ This doesn't seem correct. Don't we expect a full list of modules here? Suggest we assert on first line containing "Name", "ID" and Description" then there being more than 20 further lines (somewhat arbitrary, but we have at least that many now).